plugins/slack_webhooks/lib/samson_slack_webhooks/slack_webhooks_service.rb

```diff
@@ -36,10 +36,13 @@ def deliver_message_via_webhook(webhook:, message:, attachments:)
       payload[:channel] = webhook.channel unless webhook.channel.blank?
       payload[:attachments] = attachments if attachments.present?
 
-      Faraday.post(webhook.webhook_url, payload: payload.to_json)
-    rescue Faraday::ClientError => e
-      Airbrake.notify(e)
-      Rails.logger.error("Could not deliver slack message to webhook #{webhook.webhook_url}: #{e.message}")
+      begin
```

Comment:
why bother with this scoped `begin`? Seems unlikely an exception would be thrown in the above 3 lines?